src/xrCore/Threading/ThreadUtil.cpp

```diff
@@ -39,10 +38,21 @@ void SetThreadName(ThreadHandle threadHandle, pcstr name)
     }
 }
 
+void SetThreadName(ThreadHandle threadHandle, pcstr name)
```

Comment:
I think, we don't even need this function, SetCurrentThreadName should be enough :)

Reply:
I was thinking about that too, but decided to leave it in case someone will need it.